### DISCUSSION TITLE
Make Vue a peer dependency in the `@formkit/vue` package

### DISCUSSION THREAD

**Discussion by ScottAwesome** [formkit/formkit#917] [Ideas]
Hey all! Great library! I really enjoy it and have been using it since it dropped, more or less :)

One thing I have noticed is that `@formkit/vue` has a direct dependency listed for `vue`, as opposed to a peer dependency.

It would be awesome if we could shift to making it a peer dependency as to avoid any potential issues with dependency duplication

Thanks!

**Comment by justin-schroeder** [comment]
We actually recently switched from it being a peer dependency to it being a direct dependency. FormKit as of 0.18.0 requires Vue >= 3.3.x, we wanted to ensure that upgrade path made a hard requirement of that version (or higher) as people upgraded. I’m not entirely opposed to going back to a peerDependency but perhaps you could describe the kids of duplications you are concerned about — I would think a permissive version constraint like `"vue": "^3.3.0"` would be wide enough to not cause issues?
